docs/configuration.md

```diff
@@ -1136,6 +1136,12 @@ combination with PAPERLESS_CONSUMER_BARCODE_UPSCALE bigger than 1.0.
 
     Defaults to "300"
 
+## Audit Trail
+
+#### [`PAPERLESS_AUDIT_ENABLED=<bool>`](#PAPERLESS_AUDIT_ENABLED){PAPERLESS_AUDIT_ENABLED}
+
+: Enables an audit trail for each document,document type, correspondent, and tag. This records all changes made to the documents to meet requirement such as GoDB. **Once enabled cannot be disabled!!**
```

Comment:
1. Is this actually true about GoDB? See https://github.com/paperless-ngx/paperless-ngx/discussions/1880
2. I am not all that comfortable putting this in our docs without verifying it, and verifying it well. I never even heard of this until recently but my understanding is that it is a legal requirement in some cases, I do not want to include a statement like this without due diligence. Overall, I'd suggest removing the claim and making it something more generic.

Reply:
Yeah It's a fair point about mentioning something specific like that. I would then change to be something more generic about traceibility or.